init.lua

```diff
@@ -25,11 +25,29 @@ end
 -- `frame` support
 local use_frame = minetest.get_modpath("frame")
 
-local default_stone_sounds = default.node_sound_stone_defaults()
-local default_metal_sounds = default.node_sound_metal_defaults()
+local is_mcl_core_present = minetest.get_modpath("mcl_core") ~= nil
+local is_mcl_sounds_present = minetest.get_modpath("mcl_sounds") ~= nil
+local is_mcl_copper_present = minetest.registered_items["mcl_copper:copper_ingot"] ~= nil
+local stone_ingrediant = mcl_core_modpath and "mcl_core:stone" or "default:stone"
+local copper_ingrediant = mcl_core_modpath and "mcl_copper:copper_ingot" or 'default:copper_ingot'
```

Comment:
`ingredient` should have a `e`, not `a`. Please update all refrences to these variables as well.